root/statistics/NoStatistics.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics} from '../static/scripts/common/i18n/statistics';
+
+import StatisticsLayout from './StatisticsLayout';
+
+const NoStatistics = () => (
+  <StatisticsLayout fullWidth page="index" title={l_statistics('No Statistics')}>
+    <h2>{l_statistics('No Statistics')}</h2>
+    <p>
+      {l_statistics('Statistics have never been collected for this server. If you are the \
```

Comment:
Localized string should be on a single line.

Reply:
This isn't needed anymore, right?

Reply:
Right! It could still be replaced with ES6 template strings (supported since https://github.com/metabrainz/musicbrainz-server/commit/6ff25ea0366decc36274e9e45a635cd204a228b5).